controllers/apiv3/api_base_controller.py

```diff
@@ -122,9 +122,21 @@ def _validate_tba_auth_key(self):
         else:
             auth = ApiAuthAccess.get_by_id(x_tba_auth_key)
             if auth and auth.is_read_key:
-                self.auth_owner = auth.owner.id()
+                self.auth_owner = auth.owner
                 self.auth_description = auth.description
                 logging.info("Auth owner: {}, X-TBA-Auth-Key: {}".format(self.auth_owner, x_tba_auth_key))
             else:
                 self._errors = json.dumps({"Error": "X-TBA-Auth-Key is invalid. Please get an access key at http://www.thebluealliance.com/account."})
                 self.abort(400)
+
+class ApiSuggestBaseController(ApiBaseController):
+    def post(self, *args, **kw):
+        super(ApiSuggestBaseController, self).get(*args, **kw)
```

Comment:
So, we don't want the outputs of this to be cached - both in `memcache` on App Engine and with the `Cache-Control` headers in the response. This will happen by default here - take a look at [`CacheableHandler`](https://github.com/the-blue-alliance/the-blue-alliance/blob/master/controllers/base_controller.py#L22). 

If you want to continue sharing the same code as the `GET` handler, add a class variable flag to control when we call `super(ApiBaseController, self).get(*args, **kw)`, or duplicate that code here (but without the `super` call).

@fangeugene any preferences as to how we handle this?